ETC.tsv

```diff
@@ -97,109 +97,109 @@ ETC_20150317_000096	$Although their bright color makes them pretty visible, it's
 ETC_20150317_000159	$Madakia
-ETC_20150317_000160	Court sorcerers created the Madakia to fend off intruding grave robbers.
+ETC_20150317_000160	$Court wizards created the Madakia to fend off intruders in case they are not mere grave robbers.
 ETC_20150317_000161	$Wheelen
-ETC_20150317_000162	In order to watch over sealed ruins that lacked an overseer, monsters were created with a diversity of magic. The wheelen is one of these monsters.
+ETC_20150317_000162	$The Wheelen is one of the magical creations made to protect sealed ruins without an overseer.
 ETC_20150317_000163	$Wheelen 3
 ETC_20150317_000164	$Mauros
-ETC_20150317_000165	The stone Calvary of the Royal Mausoleum are magically assembled to withstand even the heavily armed knight.
+ETC_20150317_000165	$The stone calvary of the Royal Mausoleum are the magical creations to withstand even the heavily armed knight.
 ETC_20150317_000166	$Truffle
-ETC_20150317_000167	Though the reason is unknown, this monster has an extreme hatred towards humans. It loves to attack them for food and survival.
+ETC_20150317_000167	$Though the reason is unknown, this monster has an extreme hatred towards humans. It prefer attacking them to hunting its prey or pursuing its survival.
 ETC_20150317_000168	$Treepellia
 ETC_20150317_000169	$Bramble
 ETC_20150317_000170	Bramble Ego
 ETC_20150317_000171	$Willow
 ETC_20150317_000172	$Willowna
 ETC_20150317_000173	$Hanaming
-ETC_20150317_000174	Fallen Hanaming leaves usually don't wither rapidly.
+ETC_20150317_000174	$Leaves fallen from a dead Hanaming wither rapidly.
 ETC_20150317_000175	$Chupacabra
-ETC_20150317_000176	Their numbers grew after the Great Plant Cataclysm; now they tend to compete with humans for territory.
+ETC_20150317_000176	$Their numbers grew after the Great Plant Cataclysm; now they tend to compete with humans for territory.
 ETC_20150317_000177	$Gray Chupacabra
 ETC_20150317_000178	$Large Gray Chupacabra
-ETC_20150317_000179	Chupacabras move secretly. Based on this, it is more dangerous for humans if there are elite Chupacabras.
+ETC_20150317_000179	$Chupacabras move secretly. Based on this, it is more dangerous for humans if there are elite Chupacabras.
 ETC_20150317_000180	$Yellow Chupacabra
 ETC_20150317_000181	$Popolion
-ETC_20150317_000182	Some say a Popolion's voice can even break glass bottles.
+ETC_20150317_000182	$Some say a Popolion's voice can even break glass bottles.
 ETC_20150317_000183	$Sauga
 ETC_20150317_000184	$Saugas
 ETC_20150317_000185	$Scorpion
 ETC_20150317_000186	$Tontulia
-ETC_20150317_000187	It is a mutated monster for Tontulia to be more active.
+ETC_20150317_000187	$the Tontus mutated into Tontulia to acquire mobility.
 ETC_20150317_000188	$Tontus
-ETC_20150317_000189	The lives of residents in the dry and harsh environment were further impacted when the cacti mutated to Tontus.
+ETC_20150317_000189	$As cacti mutated into Tontus the lives of the residents in dry and harsh environments worsened.
```

Comment:
"The Tontus, having mutated from cacti, have left the residents of dry and harsh environments in a worse state of affairs."

Tontus is the subject of the sentence again. Rewritten as well.